lib/ClangImporter/ImportDecl.cpp

```diff
@@ -7979,8 +7981,11 @@ void ClangImporter::Implementation::addMemberAndAlternatesToExtension(
       return;
   
   // Then try to import the decl under the specified name.
-  if (auto *member = importDecl(decl, nameVersion))
-    addMemberAndAlternatesToExtensionDecl(member, ext);
```

Comment:
Is this call site the only one that needs modification?

Reply:
Not sure what you mean. I didn't see these comments soon enough--sorry! The smoke test failed & I did an even more cautious bit-by-bit refactoring. Didn't do quite as much as before, so I think this call site stayed the same. I tracked down the failure to the base I used before any of my changes. (Argh!) And now I'm not sure what you were referring to, sigh.
Thanks for taking the time to look at these!